templates/zerver/help/add-or-remove-users-from-a-stream.md

```diff
@@ -31,6 +31,19 @@ to a stream][configure-invites].
       To add users in bulk, you can copy members from an
       existing stream or [user group](/help/user-groups).
 
+### Mentioning a user in the compose box (alternate method)
+
+When you [mention a user](/help/mention-a-user-or-group) while
+composing a message, if they are not subscribed to the stream an alert
+will appear above the compose box. If you have permission to subscribe
+other users to the stream, the alert will contain a button allowing you
+to do so.
```

Comment:
Playing around with the text, how about something like this?

---

When you [mention a user](/help/mention-a-user-or-group) while composing a
message, an alert banner appears above the compose box if they are not
subscribed to the stream.

Click the "Subscribe them" button on the banner to add the user to the stream.
You will not see the button if you don't have permission to subscribe the user.